app/src/middleware/authorization.js

```diff
@@ -54,6 +54,7 @@ const currentObject = async (req, _res, next) => {
     }
   } catch (err) {
     log.warn(err.message, { function: 'currentObject' });
+    return new Problem(404, { detail: `Object ${req.params.objId} not found` }).send(_res);
```

Comment:
need to discuss with team if we want to return 403 or 404

Reply:
403 if I'm understanding right. Since it would throw if you provide a object ID and then that isn't found in the DB, so we wouldn't want to expose the not found as a 404 (also other errors could happen that don't necessarily specify a Not Found)

Reply:
i'm now thinking we just keep the blanket 403 for now